solana/gateway-ts/src/types/index.ts

```diff
@@ -27,6 +27,18 @@ export class GatewayToken {
     return this.state === State.ACTIVE && !this.hasExpired();
   }
 
+  equals(other: GatewayToken): boolean {
```

Comment:
This is a very OOO way of programming and I'm not sure we need it anywhere APART from tests. Therefore I'd propose to remove it. (or move it into a test utility)

Why would a library consumer ever use that equals? (e.g. either he looks at the SAME token at the same address (and only compare that) OR it is very unlikely that expireTime ever matches.

Reply:
Workaround for Chai's to.equal method. No longer necessary after updating test file